### PR TITLE
Add LightningLite to README

### PR DESCRIPTION
**Pull request by rohitgr7**
## What does this PR do?

Requested by @williamFalcon on slack.
Add lite to docs. See the change here:

https://github.com/PyTorchLightning/pytorch-lightning/tree/readme/lite#lightning-lite

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

Part of #1

cc @borda

### CODE REVIEW

**Review by Borda — APPROVED** [review]

**Review by kaushikb11 — APPROVED** [review]

File: README.md
Comment (tchaton) [inline README.md:406]:
```suggestion
- 590+ active community contributors.
```

**Review by tchaton — APPROVED** [review]
LGTM !

File: README.md
Comment (tchaton) [inline README.md:357]: This seems a bit too big.

Comment (akihironitta) [reply README.md:357]: @rohitgr7 Maybe we could remove the image (because it has nothing meaningful really) and add a minimal code snippet showing the diff between a normal PyTorch script and one with Lite to show how easy it is to scale their training.

Comment (rohitgr7) [reply README.md:357]: I'll reduce the size.

@akihironitta the banner is added to attract attention. The diff might be too long so I linked the blogpost for a better read.

**Review by justusschock — APPROVED** [review]

File: README.md
Comment (carmocca) [inline README.md:354]: Remember this needs @williamFalcon approval :)

Comment (rohitgr7) [reply README.md:354]: Yes, have pinged him. Won't merge until then.

**Comment by rohitgr7** [comment]
@williamFalcon mind review lite doc in readme?

**Pull request merged**
